tests/integration/components/two-way-data-binding-test.js

```diff
@@ -52,6 +52,36 @@ describeComponent(
           expect(this.blur).to.equal('dodge');
         });
       });
+    });
+
+    describe("default value of null with two way data binding disabled", function() {
+      beforeEach(function() {
+        this.set('make', null);
+        this.set('selectAction', (value) => {
+          this.set("make", value);
+          this.set("wasCalled", true);
+        });
+
+        this.render(hbs`
+          {{#x-select value=make one-way=true action=(action selectAction)}}
```

Comment:
Do you need to use the `action` helper here? Seem like you might be able to just bind directly via `action=selectAction`